src/test/java/org/icgc/argo/program_service/services/EgoServiceIT.java

```diff
@@ -142,6 +143,34 @@ void setUp() {
     assertThat(adminPermissions).allMatch(permission -> permission.getAccessLevel().equals("WRITE"));
   }
 
+  @Test
+  public void updateUser(){
+    val user = egoService.getUser("Test_update_user_PS@dummy.com");
```

Comment:
should not depend upon something existing in QA. If ego-qa database gets wiped clean, the integration tests should not fail. This means, the integration test should be responsible for creating and/or getting the user from ego-qa. Ego-qa should be treated as a "fresh slate" everytime the integration tests are run.